gettsim/pre_processing/policy_for_date.py

```diff
@@ -49,61 +50,146 @@
 
 
 def get_policies_for_date(policy_date, groups="all"):
-    if isinstance(policy_date, str):
-        policy_date = pd.to_datetime(policy_date)
+    """Get the state of the policy system for a particular date.
+
+    The function returns time dependent policy reforms (functions) as well as the
+    policy parameters for the current policy system.
+
+    Parameters
+    ----------
+    policy_date : datetime.date, str, int
+                  The date for which the policy system is set up.
+    groups : list, str
+             The group or a list of groups which parameters are loaded. Default is
+             all parameters
+
+    Returns
+    -------
+    params_dict : dict
+                Dictionary of parameters grouped in policy system compartments given
+                in groups.
+    policy_func_dict : dict
+                Dictionary of time dependent policy reforms. Keys are the
+                variable names they create.
```

Comment:
Please align the docstring with the NumPy format. Descriptions of parameters are indented with 4 spaces.

```suggestion
    policy_date : int, str, datetime.date, pandas.Timestamp
        The date for which the policy system is set up.
    groups : list, str
        The group or a list of groups which parameters are loaded. Default is all
        parameters

    Returns
    -------
    params_dict : dict
        Dictionary of parameters grouped in policy system compartments given in groups.
    policy_func_dict : dict
        Dictionary of time dependent policy reforms. Keys are the variable names they
        create.
```